### DISCUSSION TITLE
Site throws a 500 error whens saving the "infinity-search" theme

### DISCUSSION THREAD

**Discussion by micheledurante** [searx/searx#2695] [Public Searx instances: report a bug]
**Version of Searx, commit number if you are using on master branch and stipulate if you forked Searx**

**How did you install Searx?**

**What happened?**
Saving the theme "infinity-search" reloads the home page and returns a 500.

**How To Reproduce**
- Navigate to https://zooom.io/
- Open "Preferences" -> "User Interface"
- Select the theme "infinity-search"
- Save

**Expected behavior**
The home page loads correctly with the new theme selected.

**Screenshots & Logs**
![image](https://user-images.githubusercontent.com/17530463/112767841-37390d00-9019-11eb-8490-55446290a71d.png)
![image](https://user-images.githubusercontent.com/17530463/112767768-dc071a80-9018-11eb-946d-f9fdd4f76391.png)

**Additional context**
- Seen on Firefox version 87.0 (64-bit)
- Seen on Firefox Developer Edition version 88.0b3 (64-bit)
- The only way to recover the site is to clear the cookies

**Comment by unixfox** [comment]
There is no `infinity-search` theme on the official Searx, it's probably a custom theme from this specific Searx instance. The available themes are oscar and simple.
I moved this bug report to a GitHub discussion because it's not a bug related to the official code of Searx.

**Reply by micheledurante** [reply]
Indeed it is, sorry got confused by the the GitHub link on the instance.

Same for https://github.com/searx/searx/discussions/2698.

:)
